docs/reference/setup/restart-cluster.asciidoc

```diff
@@ -170,8 +168,12 @@ the datafeeds from {kib} or with the <<ml-open-job,open jobs>> and
 [[restart-cluster-rolling]]
 === Rolling restart
 
-
-include::{es-repo-dir}/setup/restart-cluster.asciidoc[tag=disable_shard_alloc]
+. *Disable shard allocation.*
++
+--
+include::{es-repo-dir}/upgrade/disable-shard-alloc.asciidoc[]
+This step is only necessary for <<data-node,data nodes>>.
```

Comment:
nit: it's not really necessary at all, it's just a recommendation -- see next comment with alternative suggestion. Also I think we should tweak the "Reenable allocation" wording to recognise that allocation might not have been disabled in the first place.